runtime/src/snapshot_utils.rs

```diff
@@ -179,19 +220,104 @@ pub fn package_snapshot<P: AsRef<Path>, Q: AsRef<Path>>(
     archive_format: ArchiveFormat,
     snapshot_version: SnapshotVersion,
     hash_for_testing: Option<Hash>,
-) -> Result<AccountsPackagePre> {
-    // Hard link all the snapshots we need for this package
+) -> Result<AccountsPackagePre>
+where
+    P: AsRef<Path>,
+    Q: AsRef<Path>,
+{
+    info!(
+        "Package snapshot for bank: {} has {} account storage entries",
```

Comment:
nit: `Package snapshot -> Package full snapshot`

Reply:
Same naming questions from here [1] on how to name snapshots.

[1] https://github.com/solana-labs/solana/pull/18504#discussion_r668915990

Reply:
I think the "full_" vs "incremental_" is good for clarity, thanks!